.github/workflows/python-check.yml

```diff
@@ -0,0 +1,37 @@
+name: AI and string tables checks
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.9']
```

Comment:
It gets `No steps defined in `steps` for the following jobs: build` error. Maybe it should use options without quotes?